waku/waku_api/rest/doc/relayapi_messages.yaml

```diff
@@ -0,0 +1,51 @@
+# /relay/v1/messages/{pubsubTopic}:  # Note the plural in messages
+get: # get_waku_v2_relay_v1_messages
+  summary: Get the latest messages on the polled topic
+  description: Get a list of messages that were received on a subscribed PubSub topic after the last time this method was called.
+  operationId: getMessagesByTopic
+  tags:
+    - relay
+  parameters:
+    - in: path
+      name: pubsubTopic   # Note the name is the same as in the path
+      required: true
+      schema:
+        type: string
+      description: The user ID
+  responses:
+    '200':
+      description: The latest messages on the polled topic.
+      content:
+        application/json:
+          schema:
+            $ref: './schemas/apitypes.yaml#/RelayGetMessagesResponse'
```

Comment:
I am wondering if on the server side any chunking is done in case messages are too many and each message is large in size. 
AFAIK 5MB is the limit for REST API response size.